packit_service/worker/handlers/distgit.py

```diff
@@ -361,6 +362,14 @@ def run_for_target(
             )
             logger.debug("Downstream PR created successfully.")
             model.set_downstream_pr_url(downstream_pr_url=downstream_pr.url)
```

Comment:
do we need to still set the URL separately?

Reply:
We should not need it anymore. But I didn't remove it to not make this PR even more complex. Because I think this will touch even more code. But if we want I can completely remove it.

Reply:
understood, let's do it as a followup then